portfolio.html

```diff
@@ -14,27 +14,34 @@
   </head>
   <body>
     <section class="coverPage">
-    <header class="topHeader">
-      <a href="#" class="logo">TD</a>
-      <nav class="menu">
-        <a href="#" class="toggleNav">
-          <img src="img/Union.png" alt="menu_button" />
-        </a>
-      </nav>
-    </header>
-    <br />
-    <br />
+      <header class="topHeader">
+        <a href="#" class="logo">TD</a>
+        <nav class="menu">
+          <div>
+          <a href="#" class="toggleNav">
+            <img src="img/Union.png" alt="menu_button" />
+          </a>
+        </div>
+          <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+        </nav>
+      </header>
+      <br />
+      <br />
     <section class="backgroundImg">
-      <div class="Personal_info">
-        <h1 class="introName">I'm Tamara Dimov Glad to see you!</h1>
-        <p class="intro">
+      <div class="Personal_info coverInfo">
+        <h1 class="introName coverName">I'm Tamara Dimov Glad to see you!</h1>
+        <p class="intro coverIntro">
```

Comment:
- [OPTIONAL] I would suggest you reduce the space between the headline and `Let's connect` button as to produce a close replica of the figma design. Kindly take a look at the two design, compare the space and fix the space issue on your design.

### Your Design
![your-headline](https://user-images.githubusercontent.com/26597432/206689947-3afb5d34-cddd-4076-ac0a-d0f15511e15e.png)

### Figma Design 
![figma-headline](https://user-images.githubusercontent.com/26597432/206689981-c37375a8-fe9c-4a60-ae03-38f08d6a51c9.png)